client/signup/main.jsx

```diff
@@ -715,51 +704,33 @@ class Signup extends React.Component {
 
 		const showProgressIndicator = 'pressable-nux' === this.props.flowName ? false : true;
 
+		const isReskinned = config.isEnabled( 'signup/reskin' );
+
 		return (
-			<ProvideExperimentData
-				name="refined_reskin_v2"
-				options={ { isEligible: 'onboarding' === this.props.flowName } }
-			>
-				{ ( isLoading, experimentAssignment ) => {
-					if ( isLoading ) {
-						debug( 'Waiting for experiment status to load' );
-						return null;
-					}
-
-					this.isReskinned = 'treatment' === experimentAssignment?.variationName;
-					this.isReskinned
```

Comment:
Replied about the switch: https://github.com/Automattic/wp-calypso/pull/53197/files#r652388905
Then, instead of getting the value for `isReskinned` from the experiment data, we're using a static feature flag.